confirmation/models.py

```diff
@@ -60,9 +61,10 @@ def get_object_from_key(confirmation_key: str,
         raise ConfirmationKeyException(ConfirmationKeyException.EXPIRED)
 
     obj = confirmation.content_object
-    if hasattr(obj, "status"):
-        obj.status = getattr(settings, 'STATUS_ACTIVE', 1)
-        obj.save(update_fields=['status'])
+    if activate_object:
+        if hasattr(obj, "status"):
```

Comment:
we stylistically prefer``if hasattr(obj, "status") and activate_object`` rather than nested ifs.